src/refs.bib

```diff
@@ -1744,12 +1744,20 @@ @scri.ac.uk.
     eprint = {https://academic.oup.com/bioinformatics/article-pdf/25/11/1422/944180/btp163.pdf},
 }
 
-@inproceedings{ctapipe,
-  title={Prototype Open Event Reconstruction Pipeline for the Cherenkov Telescope Array},
-  author={N\"othe, Maximilian and Kosack, Karl and  Nickel, Lukas and Peresano, Michele},
-  usera={CTA Consortium},
-  booktitle={Proceedings, 37th International Cosmic Ray Conference},
-  number={744},
-  year={2021},
-  doi={10.22323/1.395.0744},
+@ARTICLE{ctapipe,
```

Comment:
That is not a fix. You replaced the original citation to the conference proceeding with an arxiv url, even changing to article.  Why?

Reply:
Oh no. Do we have to revert?

Reply:
I can open another PR, but if there was a problem with the citation, it should be fixed properly, not by changing to the arxiv version.